apps/gallery/index.html

```diff
@@ -57,7 +57,6 @@
     <link rel="stylesheet" type="text/css" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css">
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/gaia-icons.css" />
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/bidi-helper.css" />
-    <script defer src="shared/elements/gaia-header/dist/gaia-header.js"></script>
```

Comment:
I am not sure if commenting lazy loaded file in index.html is still a requirement for build e.g. info.js is lazy loaded but its not listed in index.html. But I will definitely verify it and update.